Git Formats/Git Common.sublime-syntax

```diff
@@ -8,6 +8,8 @@ scope: text.git.common
 
 variables:
   comment_char: '[#;]'
+  hash: \b\h{7,}\b
+  repository_ref: '(?x: \b(?: [\w-]+ (/) )? [\w-]+)'  # user (optional) / repo
```

Comment:
Leave expanded? Or tighten into `(?:\b(?:[\w-]+(/))?[\w-]+)`? Also, do you want more `\b`s, or is it okay to begin/end a repo or user with `-`?

Reply:
Seems like repos can have `-` anywhere. 
![2020-07-21_18-57-42](https://user-images.githubusercontent.com/931051/88084082-27148800-cb84-11ea-9230-9eae529fa856.png)

I don't see a need to compact the regex. I would want to wrap it in a non-capturing group anyway, so I might as well use a couple spaces in there.